project/src/main/java/com/google/sps/data/VolunteeringOpportunity.java

```diff
@@ -0,0 +1,55 @@
+package com.google.sps.data;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public final class VolunteeringOpportunity {
+  private String name;
+  private Set<String> skillsRequired;
+  private int numSpotsLeft;
+  private Set<User> volunteers;
+
+  public VolunteeringOpportunity(
+      String name, Set<String> requiredSkills, int numSpotsLeft, Set<User> volunteers) {
+    this.name = name;
+    this.skillsRequired = skillsRequired;
+    this.numSpotsLeft = numSpotsLeft;
+    this.volunteers = new HashSet<User>();
+  }
+
+  public String getName() {
+    return this.name;
+  }
+
+  public void setName(String name) {
+    this.name = name;
+  }
+
+  public Set<String> getRequiredSkills() {
+    return this.skillsRequired;
+  }
+
+  public void setRequiredSkills(Set<String> requiredSkills) {
+    this.skillsRequired = requiredSkills;
+  }
+
+  public void addRequiredSkill(String requiredSkill) {
+    this.skillsRequired.add(requiredSkill);
+  }
+
```

Comment:
also maybe a deleteRequiredSkill() for if the event host no longer wants volunteers for something

Reply:
Done.

Reply:
Done.